.gitignore

```diff
@@ -1,3 +1,4 @@
 **/target
 **/*.rs.bk
 Cargo.lock
+include-file.h
```

Comment:
why ignore this file? Is this the temporary file you have planned to replace INCLUDE when the env variable is set? I think the temporary file should show up as untracked by git when the person creates it for their temporary setup.